src/main/java/org/reactome/addlinks/fileprocessors/ensembl/EnsemblBioMartFileProcessor.java

```diff
@@ -0,0 +1,167 @@
+package org.reactome.addlinks.fileprocessors.ensembl;
+
+import org.reactome.addlinks.EnsemblBioMartUtil;
+import org.reactome.addlinks.fileprocessors.FileProcessor;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.*;
+import java.util.stream.Stream;
+
+public class EnsemblBioMartFileProcessor extends FileProcessor<Map<String, List<String>>> {
+
+    public EnsemblBioMartFileProcessor()
+    {
+        super(null);
```

Comment:
For consistency, tis could be changed to `super();`.

Reply:
Just to nitpick... ;)
```java
super(null);
```
is what's already used in many other places.

Reply:
:laughing: It looked like the classes in this PR were removing the null being passed.

Reply:
Yea that ones I created I have been changing to `super()`. I can change all in add-links though to save some effort if you want (@SolomonShorser-OICR )

Reply:
I don't see a reason to start updating all the other classes, you can leave it alone. ;)